
iteration_rate: 60
device_path: '/dev/zed2'
